src/lib/styles/components/_header.scss

```diff
@@ -0,0 +1,16 @@
+header {
+  ul {
+    list-style: none;
+  }
+  li {
+    display: inline;
+    padding: 1rem;
+  }
+  a {
+    text-decoration: none;
+  }
+  button {
+    border-radius: 1em;
+    padding: 0.8em;
+  }
+}
```

Comment:
This file doesn't make much sense.

I believe it would be better to place these styles directly on the Header component.

Remember that this /components scss folder is to place reusable styles for repetitive elements we see across our UI (buttons, containers, sections, forms, etc).

But the header is not repetitive, we only have one header and we are working on it in the Header.svelte component. So it would be more practical to have all the styles and functionality related to the Header in only one place..